analyzer/config/hcl2/hcl2.go

```diff
@@ -0,0 +1,59 @@
+package hcl2
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/open-policy-agent/conftest/parser/hcl2"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/fanal/analyzer/config"
+	"github.com/aquasecurity/fanal/types"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&hcl2ConfigAnalyzer{
+		parser: &hcl2.Parser{},
+	})
+}
+
+const version = 2
```

Comment:
does version to analyzer's version? I might need to change this.

Reply:
Yes, it should be an analyzer's version. Start from 1.